modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -72,6 +72,7 @@ An additional role is required if the installation program is to create a vSpher
 `Datastore.AllocateSpace`
 `Datastore.Browse`
 `Datastore.FileManagement`
+`InventoryService.Tagging.ObjectAttachable`
```

Comment:
I checked that permission is added starting from 4.9, it would not be added here on 4.6.

Reply:
Removed